packages/clerk-js/src/ui/elements/Navbar.tsx

```diff
@@ -172,10 +176,10 @@ const NavbarContainer = (
       </Col>
 
       <Card.ClerkAndPagesTag
-        sx={theme => ({
+        sx={{
           width: 'fit-content',
-          paddingLeft: theme.space.$3,
```

Comment:
❓ Wouldn't the removal of the padding cause issues when the devmode notice is not shown?